src/services/openletters/index.jsx

```
@@ -0,0 +1,332 @@
+          sub_category: Yup.string().required("Sub Category is required"),
+          image: Yup.string().required("Image is required"),
+        })}
+        onSubmit={handleAddPicTour}
+      >
+        {({ handleSubmit, values, handleChange, setFieldValue }) => (
+          <div className="flex-col-start gap-5">
+            <div className="w-full flex gap-5 items-center justify-center">
+              <div
+                className={`relative capture-container`}
+                onClick={() => imageRef.current.click()}
+              >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)